api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -215,6 +233,10 @@ public static Timestamp deriveExpirationTimestamp(Timestamp completionTime, Long
     return new Timestamp(completionTime.getTime() + expiryDaysInMs);
   }
 
+  // Do we display this module in the user's Profile.accessModules ? This differs temporarily from
+  // whether the module is *required* in the environment because we have two Feature Flags for RAS.
+  // When the RAS roll-out is complete, we can collapse these two methods again.
+
   private boolean isModuleEnabledInEnvironment(AccessModule module) {
```

Comment:
another way to make the two `isEnabled` and `isRequired` not too far away from each other is 
```
private boolean isModuleEnabledInEnvironment(AccessModule module) {
  boolean isRequired = isModuleRequiredInEnvironment(module);
  final AccessConfig accessConfig = configProvider.get().access; 
  switch (module) {
    case RAS_LINK_LOGIN_GOV:
       return accessConfig.enableRasLoginGovLinking || isRequired;
     default:
       return isRequired;
  }
}
```